packages/test/package.json

```diff
@@ -25,7 +25,10 @@
   "author": "Roey Berman <roey@temporal.io>",
   "license": "MIT",
   "dependencies": {
-    "@opentelemetry/exporter-collector-grpc": "^0.25.0",
+    "@opentelemetry/core": "^1.0.1",
```

Comment:
No need to duplicate the dependencies that are already specified in the root package.